scripts/clowder/quay.sh

```diff
@@ -7,16 +7,56 @@ if [[ -z $QUAY_ROOT ]]; then
   exit
 fi
 
+if [[ "$(uname)" == "Darwin" ]]; then
+  MAC_OS=true
+else
+  MAC_OS=false
+fi
+
+PODMANAGER="podman"
+
+if [[ -n $2 ]]; then
+  PODMANAGER=$2
```

Comment:
ENV variable will be better than 2nd param (or argument like `--builder=docker` but it's more complicated)

so `env IMAGE_BUILDER=docker ./quay.sh sources-api` will look better

Reply:
👍